packages/next/next-server/server/api-utils.ts

```diff
@@ -302,7 +304,7 @@ export function tryGetPreviewData(
   try {
     cookies = getCookies()
   } catch {
-    // TODO: warn
+    console.warn('Did not get data from NextApiRequestCookies')
```

Comment:
This warning is not clear for end users.

Reply:
Replace warning message for end users.
https://github.com/vercel/next.js/pull/21148/commits/119d1d23a91f4c3f9f79cee43c5a9821f68161bb